@here/harp-mapview/lib/MapView.ts

```diff
@@ -2108,6 +2112,19 @@ export class MapView extends THREE.EventDispatcher {
         }
     }
 
+    private getHeightAboveTerrain() {
+        let height = Math.abs(this.projection.groundDistance(this.m_camera.position));
+
+        if (this.elevationProvider !== undefined) {
+            const heightAboveTerrain = this.elevationProvider.getHeight(this.geoCenter);
+            if (heightAboveTerrain !== undefined) {
+                height =
```

Comment:
The reason why I don't like using "let" (in general) is that for example in this case, let's say you have:
- `this.elevationProvider !== undefined`
- and `heightAboveTerrain !== undefined`

You do:

1. `let height = Math.abs(this.projection.groundDistance(this.m_camera.position));`
2. overwrite `height` with `this.projection.unprojectAltitude(this.m_camera.position) - heightAboveTerrain;`

you do unnecessary computation of `this.projection.groundDistance` (see 1st step) and don't use it later elsewhere.

Please use `if-else` construction instead.

Reply:
Done.